h2o-hadoop/scripts/docker/common/custom_startup/40_2_init_h2o

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -e
+
+if [ -f /home/h2o/h2o*${DISTRIBUTION}${VERSION}*/h2odriver.jar ]; then
+    su - h2o -c "cd h2o*${DISTRIBUTION}${VERSION}* && echo \"Launching H2O from \$(pwd)...\""
+    su - h2o -c 'hadoop jar h2o*${DISTRIBUTION}${VERSION}*/h2odriver.jar -nodes 1 -mapperXmx 1g -output jenkins_tests -disown'
```

Comment:
Is it possible to test 3nodes?

Reply:
It should be possible as long there is enough memory for 3 yarn containers.